src/SkyApm.Diagnostics.FreeRedis/FreeRedisPluginConfig.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
```

Comment:
All code files need to add the SkyAPM copyright header. Please refer to https://github.com/SkyAPM/SkyAPM-dotnet/blob/main/src/SkyApm.Abstractions/ITracingDiagnosticProcessor.cs#L1

Reply:
> All code files need to add the SkyAPM copyright header. Please refer to https://github.com/SkyAPM/SkyAPM-dotnet/blob/main/src/SkyApm.Abstractions/ITracingDiagnosticProcessor.cs#L1

ok